src/apicalls.ts

```diff
@@ -0,0 +1,50 @@
+import {TodoData, TodoObject, TodoResponse } from './todo'
+import axios from 'axios';
+
+export const addTodo = async (requestBody: TodoObject) => {
+    try {
+      await axios.post<TodoData>(
+        "https://calm-plum-jaguar-tutu.cyclic.app/todos",
```

Comment:
Any time you have the same literal string repeated more than once I'd say you should probably put it in a constant to avoid repeating it all over the place. A typo in the string won't give you an error but if you move it to a constant and have a typo when writing the name of that constant, you'll get an error